model/src/main/scala/cromwell/pipeline/model/wrapper/VersionValue.scala

```diff
@@ -14,6 +14,7 @@ object VersionValue extends Wrapped.Companion {
   val pattern = "^[0-9]+$"
   implicit val versionNumberFormat: Format[VersionValue] = wrapperFormat
   def increment(value: VersionValue): VersionValue = create(value.unwrap + 1)
+  def resetValue: VersionValue = create(0)
```

Comment:
Wouldn't be lazy val a better option here?

Reply:
@Nikolai-Isaev  I'm not sure, because I don't understand what for. If you try explain this maybe i will change it.

Reply:
I thought that since it's value is constant-like and designed not to be changed and at the same time it can be used several times in runtime.

Reply:
I suppose that using the def will work here, since lazy variables are evaluated at runtime if I'm not mistaken.